### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by praseodym**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
 dependencies {
-	compile('org.springframework.boot:spring-boot-starter')
-	compile("org.jetbrains.kotlin:kotlin-stdlib-jre8:${kotlinVersion}")
-	compile("org.jetbrains.kotlin:kotlin-reflect:${kotlinVersion}")
-	compileOnly('org.projectlombok:lombok')
-	testCompile('org.springframework.boot:spring-boot-starter-test')
-	runtime('org.postgresql:postgresql')
-	runtime("org.hsqldb:hsqldb")
+    compile('org.springframework.boot:spring-boot-starter-data-jpa')
+    compile('org.springframework.boot:spring-boot-starter-thymeleaf')
```
Comment (MatthijsKok) [inline build.gradle]: Why do we need Thymeleaf here?

Comment (praseodym) [inline build.gradle]: Maybe we don't; I just partially reverted your dependency changes to make things compile again. I'll take another look.

Comment (MatthijsKok) [inline build.gradle]: As described in [this tutorial](https://spring.io/guides/gs/uploading-files/), Thymeleaf is required to use MultipartFiles. I have confirmed that the build fails when removing Thymeleaf, as MultipartFile couldn't be resolved any more.

Comment (praseodym) [inline build.gradle]: Actually `MultipartFile` is part of `spring-web`, so `spring-boot-starter-web` will suffice. See #3.

**Pull request merged**